services/director-v2/src/simcore_service_director_v2/modules/db/repositories/comp_tasks.py

```diff
@@ -27,30 +27,33 @@
 
 
 @log_decorator(logger=logger)
-def _get_fake_service_details(service: ServiceKeyVersion) -> ServiceDockerData:
-    if "file-picker" in service.key:
-        file_picker_outputs = {
-            "outFile": {
-                "label": "the output",
-                "displayOrder": 0,
-                "description": "a file",
-                "type": "data:*/*",
-            }
+def _get_fake_service_details(
+    service: ServiceKeyVersion,
+) -> Optional[ServiceDockerData]:
+
+    if "file-picker" not in service.key:
+        return
+
+    file_picker_outputs = {
+        "outFile": {
+            "label": "File",
+            "displayOrder": 0,
+            "description": "Chosen File",
+            "type": "data:*/*",
         }
-        file_picker_type = ServiceType.FRONTEND
-        return ServiceDockerData(
-            **service.dict(),
-            name="file-picker",
-            description="file-picks",
-            authors=[
-                Author(name="ITIS", email="itis@support.com", affiliation="IT'IS")
-            ],
-            contact="itis@support.com",
-            inputs={},
-            outputs=file_picker_outputs,
-            type=file_picker_type,
-        )
-    raise ValueError("")
+    }
+    return ServiceDockerData(
+        **service.dict(),
+        name="File Picker",
+        description="File Picker",
+        authors=[
+            Author(name="Odei Maiz", email="maiz@itis.swiss", affiliation="IT'IS")
```

Comment:
MINOR: prefer fake emails instead ... will reduce spam ;-)

Reply:
well I copy pasted from the frontend code